src/VGriveClient.vala

```diff
@@ -380,7 +395,7 @@ namespace App {
             try {
                 new Thread<int>.try ("Watch remote thread", () => {
                     while (this.is_syncing ()) {
-                        Thread.usleep (this.changes_check_period*1000000);
+                        //Thread.usleep (this.changes_check_period*1000000);
```

Comment:
As I said in #70 : 

> All this Thread.sleep are used to avoid spending all the CPU of the computer. They are two separated threads, so there must be two separated sleeps.

We shoul mantain the sleep.

Reply:
Yeah! I assumed that it was in miliseconds and did not searched for this function. My bad. I'll put back